app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,47 @@
+class VotesController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @vote = Vote.new(choice: params[:vote][:choice])
+    @vote.user = current_user
+    @vote.review_id = params[:review_id]
```

Comment:
If a review has many votes, you can lookup the review and use the association methods to create the vote:
```
@review = Review.find(params[:review_id])
@vote = @reiew.votes.build(vote_params)
```